dockstore-webservice/generated/src/main/resources/pom.xml

```diff
@@ -108,7 +108,7 @@
     <dependency>
       <groupId>io.dockstore</groupId>
       <artifactId>swagger-java-zenodo-client</artifactId>
-      <version>2.1.3</version>
+      <version>2.1.4</version>
```

Comment:
New version has created/modified dates in response. See dockstore/swagger-java-zenodo-client#25